tests/testthat/test-as_polars.R

```diff
@@ -180,9 +179,7 @@ test_that("tests for vctrs_rcrd", {
 
   expect_identical(length(as_polars_series(vec)), 2L)
 
-  # TODO: this should work
-  # https://github.com/pola-rs/r-polars/issues/575
-  # pl$DataFrame(foo = vec)
+  pl$DataFrame(foo = vec)
```

Comment:
I think you forgot to wrap this in an `expect_` statement

Reply:
Indeed.
I couldn't think of a good test to confirm this, so I decided on a snapshot test.